hypervisor/arch/x86/guest/vcpuid.c

```diff
@@ -299,7 +299,7 @@ void guest_cpuid(struct acrn_vcpu *vcpu,
 
 	/* vm related */
 	if ((leaf != 0x1U) && (leaf != 0xbU) && (leaf != 0xdU)) {
-		struct vcpuid_entry *entry =
+		const struct vcpuid_entry *entry =
 			find_vcpuid_entry(vcpu, leaf, subleaf);
```

Comment:
merge 2 lines. and check the entire patches

Reply:
Will do it